### ISSUE TITLE
[Redmine #13657] icingaweb2 is showing backend icinga2 process is not running

### ISSUE THREAD

**Issue by icinga-migration** [lazyfrosch/icinga2-test#45]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/13657

    Author: rnekkanti
    Assignee: (none)
    Status: Closed
    Target Version: (none)
    Created: 2016-12-16T22:10:54Z (closed on 2017-01-11T13:09:00Z)
    Last Update: 2017-01-11T13:09:00Z (in Redmine)

    Icinga Version: 2.6.0-1
    Backport?: Not yet backported
    Include in Changelog: 1

---

After distributed monitoring is setup, icingaweb2 is showing backend icinga2 process is not running on the master node even tough icinga2 process is clearly up and running. I am seeing this issue only after distributed monitoring is setup. When running as standalone icinga2 instance it is not showing that problem.

Process Information

Stats

Process Info
Program Version r2.6.0-1
Program Start Time 2016-12-16 21:18:24
Last Status Update 17m 38s ago
Last External Command Check 17m 38s ago
Last Log File Rotation N/A
Global Service Event Handler N/A
Global Host Event Handler N/A
Active Endpoint ops-monitor-test-1.globix-sc.gracenote.com
Backend icinga is not running

Icinga2 version 2.6.0-1
icingaweb2 version: 2.4.0
Running on Ubuntu 14.04
icinga2 is running as a service.

---

**Relations**:

* relates [#13655](https://dev.icinga.com/issues/13655)

---

**Updated by geotek on 2016-12-17T11:19:37Z**

* File added _report.1481930237.496197_

Same symptom here. After upgrading to 2.6.0 satellite runs fine but master constantly and repeatedly crashes several minutes after starting. Message is:

    systemd-coredump[9938]: Process 1550 (icinga2) of user 489 dumped core.

Please see the crash report enclosed

---

**Updated by lynxo on 2016-12-19T07:59:16Z**

* File added _icinga2_crash_repots.tgz_

Since update (2.5.4 -&gt; 2.6.0) on friday same problem.
Every time kernel segfaults.
No way to work arround, so we have to downgrade our single icinga2 node.

attachment:icinga2\_crash\_repots.tgz

---

**Updated by rnekkanti on 2017-01-09T18:17:47Z**

Any update on this ?

---

**Updated by mfriedrich on 2017-01-11T13:08:25Z**

* **Relates** set to _13655_

---

**Updated by mfriedrich on 2017-01-11T13:09:00Z**

* **Status** changed from _New_ to _Closed_

This is a duplicate of [\#13655](https://dev.icinga.com/issues/13655) and we plan to address this with 2.6.1.
